support-internationalisation/src/main/scala/com/gu/i18n/Currency.scala

```diff
@@ -7,10 +7,180 @@ sealed trait Currency {
   def iso: String
 }
 
+case class OtherCurrency(iso: String, glyph: String) extends Currency
+
 object Currency {
-  val all = List(GBP, USD, AUD, CAD, EUR, NZD, SEK, CHF, NOK, DKK)
+  val websiteSupportedCurrencies = List(
```

Comment:
We now have the hard coded currencies which we've always had plus a map of other valid iso currency codes and their associated symbols